src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -50,6 +51,31 @@
 CHECKIN_QUOTA_WINDOW = 60
 
 
+class CheckinMessage(TypedDict):
+    payload: str
+    start_time: str
+    project_id: str
+    sdk: str
+
+
+class CheckinTrace(TypedDict):
+    trace_id: str
+
+
+class CheckinContexts(TypedDict):
+    trace: NotRequired[CheckinTrace]
+
+
+class CheckinPayload(TypedDict):
```

Comment:
any thought to adding `total=False` here as a safety measure? and then removing it when we have some confidence

Reply:
I guess this shouldn't result in any run time errors right? just mypy errors. so maybe not worht it